Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.yml

```
@@ -117,6 +117,365 @@ script:
+      - environment.ts
+      - tpl 
+      - tfvars
+      - unknown 
+      - csproj 
+      - packages.config
+      - Directory.Packages.props
+      - paket.dependencies
+      - paket.references
+      - paket.lock
+      - sln
+      - config      
+      - composer.json
+      - composer.lock
+    - name: sast_labels
+      description: s label associated with Static Application Security Testing (SAST) results to filter the output.
+      auto: PREDEFINED
+      predefined:
+      - CustomPolicy
+      - CWE top 25
+      - OWASP top 10
+      isArray: false
+    - description: Whether to retrieve only fixable issues. Recommended to use this argument with another argument code_categories ,for example.
+      name: fixable_only
+      type: Boolean
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - description: Specifies the repository names to filter results from. Supports comma separated values.
+      name: repositories
+      isArray: true
+    - description: Filters based on risk factors related to secrets. Supports comma separated values.
+      name: secrets_risk_factors
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - PublicRepository
+      - PrivateRepository
+      - User
+      - Privileged
+      - Valid
+      - Invalid
+      - Unknown
+      - FoundInHistory
+    - description: Defines specific areas or categories within which to search. Supports comma separated values. When specifying `search_scopes`, `search_term` argument is required.
+      name: search_scopes
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - IacMisconfiguration
+      - IacExternalModule
+      - ImageReferencerVul
+      - ImageReferencerLicenses
+      - Vulnerabilities
+      - Licenses
+      - Secrets
+    - description: The criticality of issues. Supports comma separated values.
+      name: severities
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - INFO
+      - LOW
+      - MEDIUM
+      - HIGH
+      - CRITICAL
+    - description: Filters results based on risk factors for vulnerabilities. Supports comma separated values.
+      name: vulnerability_risk_factors
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - AttackComplexity
+      - AttackVector
+      - DoS
+      - HasFix
+      - RemoteExecution
+      - ExploitInTheWild
+      - ExploitPOC
+      - IsUsed
+      - ReachableFunction
+      - ManifestInRuntime
+      - RepositoryInRuntime
+      - PackageInRuntime
+    - description: A keyword or phrase used to narrow down results within specific scopes (like a particular vulnerability). It's needed if you are filtering results by search_scopes.
+      name: search_term
+      isArray: false
+      required: false
+    - description: Relevant only to- IacMisconfiguration,Vulnerabilities.
+      name: iac_tags
+      isArray: false
+    - description: License types to filter by. Supports comma separated values.
```

```suggestion
    - description: List of license types to filter by. Supports comma separated values.
```
add example of license types and a link to the list